mpa/modules/hooks/early_stopping_hook.py

```diff
@@ -62,12 +60,14 @@ def __init__(self,
                  rule: Optional[str] = None,
                  patience: int = 5,
                  iteration_patience: int = 500,
-                 min_delta: float = 0.0):
+                 min_delta: float = 0.0,
+                 warmup_iters: int = 0):
```

Comment:
@chuneuny-emily Is there a background in which this was added?

Reply:
It was required in mmseg default setting, but now it may not used with latest tuned parameters. After checking it, let me revert it to original one. Thanks for your feedback.